### PR TITLE
feat(settings): show QR code for two step auth

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - we need to display the QR code for users to set up their
   authenticators

This commit:
 - get and show the QR code

Closes: #4972 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).

### CODE REVIEW

**Comment by chenba** [comment]
Setting it as a draft because I realized that I missed some styles around the QR code.

File: packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx [hunk 1]
```diff
@@ -17,12 +31,38 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
...
+      setSecret(x.createTotp.secret);
+      setRecoveryCodes(x.createTotp.recoveryCodes);
+    },
+    onError: () => {
+      alertBar.error('There was a problem retrieving your code.');
+    },
+  });
+
+  useEffect(() => {
+    createTotp({ variables: { input: {} } });
```
Comment (dannycoates) [inline packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx:55]: with an unverified session I get this screen...
Screen Shot 2020-10-19 at 12 37 32 PM

I think we need to put this call behind the guard somehow.

**Review by dannycoates — COMMENTED** [review]
UI looks good. Other than the unverified session issue this is r+

File: packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx [hunk 2]
```diff
@@ -17,12 +32,40 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
...
+    },
+    onError: () => {
+      alertBar.error('There was a problem retrieving your code.');
+    },
+  });
+
+  const session = useSession();
+
+  useEffect(() => {
+    session.verified && createTotp({ variables: { input: {} } });
```
Comment (chenba) [inline packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx:58]: @dannycoates made this change.  Thanks!

**Review by dannycoates — APPROVED** [review]
looks great!

**Pull request merged**